### DISCUSSION TITLE
In Elsa 3.1.3, the Switch (flow) activity did not set the Expression property of variable parameters, resulting in the inability to pass parameters to the Switch?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5699] [Design & tech]
In Elsa 3.1.3, the Switch (flow) activity did not set the Expression property of variable parameters, resulting in the inability to pass parameters to the Switch?
![switch](https://github.com/elsa-workflows/elsa-core/assets/171897414/c2d1533e-d0a5-4c8e-883b-8b47dc451723)

**Comment by glime-ai** [comment]
Based on the context provided, it seems that there is an issue with the Switch activity in Elsa 3.1.3 where the Expression property of variable parameters is not being set correctly, which would prevent parameters from being passed to the Switch activity. This could be a bug or a limitation in the Elsa 3.1.3 designer interface.

However, without specific error messages or code examples, it's difficult to provide a definitive answer or solution. If this is a bug, it may have been reported and could be addressed in a future update. It would be advisable to check the official Elsa GitHub issues page for any reported bugs related to this issue and see if there are any workarounds or fixes available.

If you are experiencing this issue, you could try the following steps:
1. Ensure that you are using the latest version of Elsa and that all dependencies are up to date.
2. Check the official Elsa documentation and GitHub issues for any known issues or updates related to the Switch activity.
3. If the issue persists, consider reporting it on the Elsa GitHub issues page with detailed information, including the steps to reproduce the problem, any error messages, and your Elsa workflow configuration.

If you can provide more details or clarify the issue further, I may be able to offer more specific guidance.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
